src/main/webapp/app/shared/category-selector/category-selector.scss

```diff
@@ -26,3 +26,21 @@ jhi-category-selector {
         }
     }
 }
+
+.category-name:hover + .tag-options {
+    transform: scale(1.5);
+    transform-origin: 50% 50%;
+}
+
+.tag-options {
+    position: absolute;
+    transition: all 0.2s ease-in-out;
+}
+.tag-options:hover {
```

Comment:
Maybe combine with `.category-name:hover + .tag-options` to avoid the duplication?

```css
.category-name:hover + .tag-options,
.tag-options:hover {
    transform: scale(1.5);
    transform-origin: 50% 50%;
}

Reply:
Done in 835e24f4cd81348ec6efe86a3d67fac46a45e8c1